### PR TITLE
added css file

### PR DESCRIPTION
**Pull request by ididmitrov**

### CODE REVIEW

File: README.md
Comment (galiabakri) [inline README.md:1]: this file is wrong

**Review by galiabakri — COMMENTED** [review]
:)

File: README.md
Comment (ididmitrov) [reply README.md:1]: Ill fix it, thanks!

Comment (DanielKQA) [reply README.md:1]: The file is correct one :-)

**Review by galiabakri — APPROVED** [review]

**Review by tripper8 — CHANGES_REQUESTED** [review]
Please make the necessary changes. Thank you.

**Pull request closed without merging**